storage-proofs/porep/src/lib.rs

```diff
@@ -1,6 +1,8 @@
 //requires nightly, or later stable version
 //#![warn(clippy::unwrap_used)]
 
+#![cfg_attr(target_arch = "aarch64", feature(stdsimd))]
```

Comment:
this means that aarch64 now requires a nightly compiler, which is fine, but should be documented

Reply:
Thanks for the quick review.

Seems the nightly compiler requirement has been documented in README.md already:
275 ## Building for Arm64
276
277 In order to build for arm64 the current requirements are
278
279 - nightly rust compiler
280
281 Example for building `filecoin-proofs`
282
283 ```
284 $ rustup +nightly target add aarch64-unknown-linux-gnu
285 $ cargo +nightly build -p filecoin-proofs --release --target aarch64-unknown-linux-gnu
286 ```

Do we need extra comments in `storage-proofs/porep/src/lib.rs` before the cfg_attr line or elsewhere? Maybe something like `//requires nightly, or later stable version to build for aarch64`.

Best Regards.

Reply:
ha, I forgot about that, perfect thank you!